docs/components/DataList/Web.stories.tsx

```diff
@@ -81,9 +81,13 @@ const Template: ComponentStory<typeof DataList> = args => {
     created: new Date(node.created),
   }));
 
+  const [selectedItems, setSelectedItems] = useState<Array<string>>([]);
+
   return (
     <DataList
       {...args}
+      selected={selectedItems}
+      onSelect={setSelectedItems}
```

Comment:
actually, we should not implement/expose this yet because the "select all" isn't there yet and this will be misaligned on the example 

![image](https://github.com/GetJobber/atlantis/assets/15986172/3a28804b-724a-4554-ae94-bf10980e2608)

Reply:
I can remove it once we get finished the review before merging. That way we can test and before approving it

Reply:
@darryltec not sure if it pinged you

Reply:
agreed! yup. I get pinged on everything!